### ISSUE TITLE
Activity: add version number to update items

### ISSUE THREAD

**Issue by keoshi** [Automattic/wp-calypso#24795]
If a plugin or a theme has new updates available, we should show the version number on the AL to both make the historical record richer and give more reassurance when customers are updating an item.

This applies to the following items (at least):

- Plugin Update Available
- Plugin Updated
- Plugin Autoupdated
- Theme Update Available
- Theme Updated
- Theme Autoupdated

We should also make sure Core updates events follow the same language pattern.

It could look something like this (bad mockup):

![image](https://user-images.githubusercontent.com/390760/39878149-88a57b12-546f-11e8-9cdc-ad70a4f96fc5.png)

But ideally should follow the styling of `activity-log-tasklist__update-version` in https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/activity-log-tasklist/style.scss#L46-L48

**Comment by simison** [comment]
@keoshi should we show the previous version number? — something like this:

> Jetpack・from 6.0.0 to 6.1.0
> Plugin Autoupdated

Or perhaps:

> Jetpack・6.0.0 → 6.1.0
> Plugin Autoupdated

**Comment by simison** [comment]
Could we first as a MVP settle down to something that just renders a version without styles? Would be faster to get it out across web+mobile.

![image](https://user-images.githubusercontent.com/87168/41900199-d05de048-7936-11e8-87c6-f5dcc6ee38a7.png)

**Comment by keoshi** [comment]
Hm, I have mixed feelings about adding the current version, but I can see the diff between them being useful to customers. I dig the latter format with the arrow, keeps it nice and tight.

@eliorivero what are your thoughts on this?

**Comment by keoshi** [comment]
I agree, @simison — could we include the version number in the link for the MVP then?

**Comment by simison** [comment]
>could we include the version number in the link for the MVP then?

Sure, like so?

![image](https://user-images.githubusercontent.com/87168/41901058-b3a7e3ca-7938-11e8-88fb-7a5cf7b0c450.png)

**Comment by keoshi** [comment]
Yes, that looks great!

**Comment by simison** [comment]
Being worked at D15322-code

**Comment by gititon** [comment]
@keoshi @eliorivero, I'm working on this. Including the version number in the link is simple enough, but what should we do for plugin update events where we go from a version to another version?

In my current PR, it says "WooCommerce updated from 2.0 to 3.4.3". I imagine that you wouldn't want that entire thing to be a link? I could omit the "from" version so that it is formatted the same way as update available, I just thought it would be nice for users to see/have that info.

What would you prefer? Thanks!

**Comment by keoshi** [comment]
For the MVP I’d include the version in the link. If we want to show the two versions, @simison’s second alternative here looks like a winner: https://github.com/Automattic/wp-calypso/issues/24795#issuecomment-400227455

**Comment by simison** [comment]
Some screenshots:

<img width="456" alt="screen shot 2018-06-26 at 22 17 39" src="https://user-images.githubusercontent.com/87168/41934480-796e37f2-798f-11e8-95be-c401937bbb0e.png">
<img width="638" alt="screen shot 2018-06-26 at 22 16 08" src="https://user-images.githubusercontent.com/87168/41934481-79a0c7bc-798f-11e8-9414-3c7d92f1c660.png">
<img width="670" alt="screen shot 2018-06-26 at 22 14 58" src="https://user-images.githubusercontent.com/87168/41934482-79c93116-798f-11e8-84b7-570bb29a8efb.png">

_(Missing actor is a known bug)_

**Comment by gititon** [comment]
Thanks, @keoshi . I deployed this, please let me know if it looks good and we can close this issue.

**Comment by keoshi** [comment]
@gititon Looking amazing on most sites I've tested, thank you!

On one of them, however, I don't see any version numbers. This change should be retroactive, right?

**Comment by gititon** [comment]
@keoshi yes, it should. Can you confirm that the additional change I made resolves it for the site you're looking at as well?

**Comment by keoshi** [comment]
@gititon Replied in Slack, but yes: it looks great now! Let's close this one, and thanks for your work on it!
